sharepoint/3.1/modules/ROOT/pages/sharepoint-connector-reference.adoc

```diff
@@ -1,1320 +1,1365 @@
+=== SharePoint Online
 
-=== Parameters
 
+==== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-|Name | String | The name for this configuration. Connectors reference the configuration with this name. | |x
-| Connection a| * Okta
-* Online
-* Security Token
- | The connection types that can be provided to this configuration. | |x
-| Expiration Policy a| ExpirationPolicy | Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime actually purges the instances when it sees fit. | |
-|===
-
-== SharePoint Online Connection Types
-
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| * <<sharepoint-online_okta, Okta>>
+* <<sharepoint-online_online, Online>>
+* <<sharepoint-online_security-token, Security Token>>
+ | The connection types to provide to this configuration. | | x
+| Charset a| Enumeration, one of:
+
+** US_ASCII
+** ISO_8859_1
+** UTF_8
+** UTF_16BE
+** UTF_16LE
+** UTF_16 |  |  UTF_8 |
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. Mule purges the instances as appropriate. |  |
+|===
+
+==== Connection Types
 [[sharepoint-online_okta]]
-=== Okta
+===== Okta
 
-==== Parameters
 
+====== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Username a| String | Username that you want to login with against Okta Identity Provider. | |x
-| Password a| String | Password for the user that you are going to login against Okta Identity Provider. | |x
-| Okta Domain a| String | Domain name for Okta Identity Provider. | |x
-| Okta API token a| String | | |x
-| SharePoint Online embedded link from Okta a| String | | |x
-| Disable SSL certificate validation a| Boolean | When dealing with HTTPS certificates, if the certificate is not signed by a trusted partner, the server might respond with an Exception.  To prevent this, it is possible to disable the CN (Common Name) check.  Note: This is not recommended for production environments. | false |
-| Proxy Settings a| One of:
-
-* proxy
-* ntlm-proxy | Settings for the proxy server that the connector is going to send data through. | |
-| Reconnection a| Reconnection | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. | |
+| Okta Username a| String |  Username to use to log in to the Okta Identity Provider |  | x
```

Comment:
Not super important but there is a mix of period and no period in the table. I think , in this case, they all need periods to follow the consistency rule.

Reply:
Not fixing these because they are all over and not that important.